.run/nodeop (aio-01).run.xml

```diff
@@ -1,5 +1,5 @@
 <component name="ProjectRunConfigurationManager">
-  <configuration default="false" name="nodeop (aio-01)" type="CMakeRunConfiguration" factoryName="Application" PROGRAM_PARAMS="--config-dir $ProjectFileDir$/etc/config/nodeop/aio --contracts-console --data-dir /tmp/wire-chain-aio-01 --genesis-json $ProjectFileDir$/etc/config/nodeop/aio/genesis.json --signature-provider &quot;SYS8FwtmL4r1z27ZR2jM8Wc4PAsLTjXk5rWz9939Ds7mJrrK8bUGo=KEY:5JtGeD8XhZkXXYcpi1KQtf1ZQY2SdkYuVS4QidDg4d6cC9qMfjy&quot;" REDIRECT_INPUT="false" ELEVATE="false" USE_EXTERNAL_CONSOLE="false" EMULATE_TERMINAL="true" WORKING_DIR="file://$ProjectFileDir$" PASS_PARENT_ENVS_2="true" PROJECT_NAME="wire-sysio" TARGET_NAME="nodeop" CONFIG_NAME="Debug-clang-18-no-cache" RUN_TARGET_PROJECT_NAME="wire-sysio" RUN_TARGET_NAME="nodeop">
+  <configuration default="false" name="nodeop (aio-01)" type="CMakeRunConfiguration" factoryName="Application" PROGRAM_PARAMS="--config-dir $ProjectFileDir$/etc/config/nodeop/aio --contracts-console --data-dir /tmp/wire-chain-aio-01 --genesis-json $ProjectFileDir$/etc/config/nodeop/aio/genesis.json --signature-provider &quot;SYS8FwtmL4r1z27ZR2jM8Wc4PAsLTjXk5rWz9939Ds7mJrrK8bUGo=KEY:5JtGeD8XhZkXXYcpi1KQtf1ZQY2SdkYuVS4QidDg4d6cC9qMfjy&quot;" REDIRECT_INPUT="false" ELEVATE="false" USE_EXTERNAL_CONSOLE="false" EMULATE_TERMINAL="true" WORKING_DIR="file://$ProjectFileDir$" PASS_PARENT_ENVS_2="true" PROJECT_NAME="wire-sysio" TARGET_NAME="nodeop" CONFIG_NAME="RelWithDebInfo" RUN_TARGET_PROJECT_NAME="wire-sysio" RUN_TARGET_NAME="nodeop">
```

Comment:
I've reviewed and set ignore for `.run` configs, especially now that we have a tool to generate them quickly.  Just an FYI @heifner